server/src/main/java/io/crate/expression/scalar/postgres/PgArrayToString.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.expression.scalar.postgres;
+
+import io.crate.expression.scalar.ArrayToStringFunction;
+import io.crate.expression.scalar.ScalarFunctionModule;
+import io.crate.metadata.FunctionName;
+import io.crate.metadata.functions.Signature;
+import io.crate.metadata.pgcatalog.PgCatalogSchemaInfo;
+import io.crate.types.DataTypes;
+
+import static io.crate.metadata.functions.TypeVariableConstraint.typeVariable;
+import static io.crate.types.TypeSignature.parseTypeSignature;
+
+public class PgArrayToString extends ArrayToStringFunction {
+    public static final String NAME = "array_to_string";
+    private static final FunctionName FQN = new FunctionName(PgCatalogSchemaInfo.NAME, NAME);
```

Comment:
Instead of creating a copy of the `ArrayToString` function we could just change the FQN at the existing function by adding the `pg_catalog` schema. As `pg_catalog` is part of the [search_path](https://crate.io/docs/crate/reference/en/4.6/config/session.html#conf-session-search-path), the function registry will look up `pg_catalog.array_to_string` if the function is only expressed by `array_to_string` and cannot be found in overloaded schemas.